tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2879,6 +2891,10 @@ describe('scalar card', () => {
           MIN_VALUE: 4.532374100719424,
           MAX_VALUE: 18.976711362032464,
           PERCENTAGE_CHANGE: 1.2427022518765645,
+          STEP_AT_MAX: 4,
+          STEP_AT_MIN: 3,
+          MEAN: 11.5,
```

Comment:
I would expect this to be 12, given the range is steps 2 to 4.

Reply:
Yup. I calculated using all the steps and so did the algorithm. My mistake. Fixed